contracts/Market.sol

```diff
@@ -14,8 +14,8 @@ import "./SignatureLib.sol";
 
 // Put these in the ERC721 contract
 struct Bet {
-	bytes32 propositionId;
-	bytes32 marketId;
+	bytes16 propositionId;
+	bytes16 marketId;
 	uint256 amount;
 	uint256 payout;
 	uint256 payoutDate;
```

Comment:
you could make payoutDate a uint48, which is plenty if it is a unix timestamp. Then, you could make the proposition Id & market Id a bytes13. This would save 1 storage slot

Reply:
Sounds like a new ticket :-)